packages/apps/human-app/server/README.md

```diff
@@ -59,7 +59,7 @@ and transformation of data. NestJS provides many built-in pipes, so check the do
 - **Filters**: Classes implementing the `ExceptionFilter` interface, used to handle exceptions.
 - **Interceptors**: Used to bind extra logic before or after method execution or to extend the behavior of the method.
 
-### Model 
```

Comment:
this was not the subject of the reviewer's comment

Reply:
fix: fa99045d